bin/update_and_install_system_packages.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Adapted from [this script](https://pythonspeed.com/articles/system-packages-docker/#debian-ubuntu-and-the-debian-based-python-base-image)